templates/docs/examples/patterns/tabs/_script.js

```diff
@@ -0,0 +1,101 @@
+(function () {
+  var keys = {
+    left: 37,
+    right: 39,
+  };
+
+  var direction = {
+    37: -1,
+    39: 1,
+  };
+
+  /**
+    Attaches a number of events that each trigger
+    the reveal of the chosen tab content
+    @param {Array} tabs an array of tabs within a container
+  */
+  function attachEvents(tabs) {
+    tabs.forEach(function (tab, index) {
+      tab.addEventListener('keyup', function (e) {
+        var key = e.keyCode;
+
+        if (key === keys.left || key === keys.right) {
+          switchTabOnArrowPress(e, tabs);
+        }
+      });
+
+      tab.addEventListener('click', function (e) {
+        e.preventDefault();
+        setActiveTab(tab, tabs);
+      });
+
+      tab.addEventListener('focus', function () {
+        setActiveTab(tab, tabs);
+      });
+
+      tab.index = index;
+    });
+  }
+
+  /**
+    Determine which tab to show when an arrow key is pressed
+    @param {KeyboardEvent} event
+    @param {Array} tabs an array of tabs within a container
+  */
+  function switchTabOnArrowPress(event, tabs) {
+    var pressed = event.keyCode;
```

Comment:
`event.keyCode` is deprecated. Ideally we should not encourage using it, and use `event.code` instead.

`keyCode` may still be needed for IE, but if it's too hard to support both I guess we are fine with skipping adding this for IE and treating it as an enhancement for newer browsers.

Reply:
That's a fair point - I've updated with support for both.